### PR TITLE
Add world generator and misc fix for crafting NPC and Item

### LINKED ISSUE #59: Port a world generator
Use save word in pr #58 but in time we need to add a world generator script.

### PR DESCRIPTION
**Pull request by Jhobean**
1- I added @Soulless-1 's world generator and fix #59 

2- Removed the Saves files folder (not need anymore)

3- Added some missing NPC and item

4- Add the Dungeons Chest by using Soulless Script (require for world spawning)

5- Added the files for manual Crop generation (People can decide to use them)

6- Added 20 OSI book template write one by one by soulless and implement the bookcase fill in the world generator

7- Started to remove some defs_heght after validating height is set in CHARDEF (Eventualy will remove all the defs_height)
 
8- update the .add db and fix some items problem

### CODE REVIEW

File: items/i_deco_dungeons_traps_dead.scp [hunk 1]
```diff
@@ -1269,33 +1269,26 @@ CATEGORY=Decoration - Dungeons and The Dead
...
-[ITEMDEF 01185]
-DEFNAME=i_gravestone_19
-NAME=memorial to Gwenno
-//TYPE=t_sign_gump
-//TDATA2=066
-FLIP=1
-WEIGHT=800.0
-RESOURCES=80 i_rock_plain
+[itemdef 01185]
+defname=memorialgwenn
```
Comment (cbnolok) [inline items/i_deco_dungeons_traps_dead.scp:1273]: Is that the correct defname?

File: web/spherestatus.htm [hunk 2]
```diff
@@ -12,11 +12,11 @@
 <p align="left"><strong><font size="5" face="Arial" color="#5B86F7">Status:<br>
 </font><font color="#5B86F7" size="1" face="Arial"><br>
 </strong>This information is updated in real time from the server every 5 minutes. Last
-update time 2020/12/17 16:40:06<br>
+update time 2021/08/13 00:11:24<br>
```
Comment (cbnolok) [inline web/spherestatus.htm:15]: Was this file intended to be committed?

**Review by cbnolok — CHANGES_REQUESTED** [review]

File: web/spherestatus.htm (hunk 2 above)
Comment (Jhobean) [reply web/spherestatus.htm:15]: No.  This file should be in the gitignore

File: items/i_deco_dungeons_traps_dead.scp (hunk 1 above)
Comment (Jhobean) [reply items/i_deco_dungeons_traps_dead.scp:1273]: It was like this on an other script pack. Should be change its not the convention. Good see. Im out for the week end. Ill check monday or if you want you can fix it

**Comment by Jhobean** [comment]
Should Be OK @cbnolok

File: crafting/crafting_events.scp [hunk 3]
```diff
@@ -88,6 +88,12 @@ if <def0.scp.NewCrafting_<serv.skill.<argn>.key>> == 1
 endif
 
 on=@skillmakeitem
+//<ACT>=l'item crafté
```
Comment (cbnolok) [inline crafting/crafting_events.scp:91]: Could you translate in english?

File: crafting/crafting_events.scp [hunk 4]
```diff
@@ -98,6 +104,12 @@ endif
 ref2 = <act.uid>
 ref2.name = <serv.itemdef.<ref2.baseid>.name>
 ctag.craft.message = Your last successful creation: <ref2.name>.
+
+//Give the color of folded cloth to the new item
+if STRMATCH(*Tailoring*,<uid.<ref2>.SKILLMAKE.1.key>) && STRMATCH(*i_cloth*,<serv.itemdef.<ref2.baseid>.resources>)
```
Comment (cbnolok) [inline crafting/crafting_events.scp:109]: Internally defnames (skills, items...) are converted to fixed a numerical value. Could you use the == operator to do the checks? Using string comparison when you could just use numerical comparison is bad practice and we shouldn't have that on default scriptpack

File: functions/f_dungeon_chest.scp [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+[function remove_dungeon_chests]
+for n 5 
+  forinstances i_dungeon_chest<dlocal.n>
+    remove
+  endfor
+endfor
+
+[function f_trap_container]	//trapcontainer,[type 1-3],[strength],[uid of the person who trapped it]
+if (!<isevent.t_container_locked>) && (!<isevent.t_container>) && (!<type>==t_container_locked) && (!<type>==t_container)
```
Comment (cbnolok) [inline functions/f_dungeon_chest.scp:26]: Can you directly use != operator?

**Review by cbnolok — CHANGES_REQUESTED** [review]

**Pull request merged**